components/FormCard.tsx

```diff
@@ -127,18 +149,35 @@ export const FormCard: React.FC<FormCardProps> = ({
     />
   ))
 
+  const textAlignment =
+    align === 'center' ? 'center' : align === 'right' ? 'end' : 'start'
+  const noBgStyles = noBg ? 'bg-transparent shadow-none border-0 py-0' : 'py-4'
+
   return (
     <div className={`row${border ? ' border' : ''}`}>
-      <div className={`card shadow-sm col-12`}>
-        <div className="card-body text-center">
-          {title && <h2 className="card-title fw-bold mb-3">{title}</h2>}
+      <div
+        className={`${noBgStyles} px-3 card shadow-sm col-12 align-items-stretch`}
+      >
+        <div
+          className={`p-0 card-body text-center align-self-${textAlignment}`}
+        >
+          {title && <h3 className={`card-title fw-bold mb-4`}>{title}</h3>}
         </div>
         <div className="text-start">{optionsList}</div>
-        {submitError && <h6 className="text-danger">{submitError}</h6>}
-        <div className="text-center mb-4">
-          <Button onClick={btnOnClick} type="primary" color="white">
-            {onSubmit.title}
-          </Button>
+        {submitError && <ErrorMessage error={submitError} />}
+        <div className={`text-center align-self-${textAlignment}`}>
+          {newBtn ? (
+            <button
+              onClick={btnOnClick}
+              className={styles.formCard__submitButton}
+            >
+              {onSubmit.title}
+            </button>
+          ) : (
+            <Button onClick={btnOnClick} type="primary" color="white">
```

Comment:
I kept the old button because I wasn't sure if it should be removed. My thought is to keep it as the default one.